.github/workflows/build-ot3-actions.yml

```diff
@@ -30,13 +30,38 @@ on:
         default: 'stage-prod'
 
 jobs:
+  decide-refs:
```

Comment:
This looks like it can be done below when we also call "decide-refs" since all we are doing is adding the variant variable.

Reply:
it needs to be done here because the subsequent job will run in different infrastructure depending on its output. however, we can remove the one below.

Reply:
(and did so)